src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderCaptureRequest.cs

```diff
@@ -1,15 +1,28 @@
-﻿using System.Collections.Generic;
-
-namespace SwedbankPay.Sdk.PaymentOrders
-{
-    public class PaymentOrderCaptureRequest
-    {
-        public PaymentOrderCaptureRequest(Amount amount, Amount vatAmount, List<OrderItem> orderItems, string description, string payeeReference)
-        {
-            Transaction = new PaymentOrderCaptureTransaction(amount, vatAmount, orderItems, description, payeeReference);
-        }
-
-
-        public PaymentOrderCaptureTransaction Transaction { get; }
-    }
+﻿using System.Collections.Generic;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    /// <summary>
+    /// API request object for capturing funds for a payment order.
+    /// </summary>
+    public class PaymentOrderCaptureRequest
+    {
+        /// <summary>
+        /// Instantiates a <see cref="PaymentOrderCaptureRequest"/> with the provided parameters.
+        /// </summary>
+        /// <param name="amount">The amount of funds to capture.</param>
+        /// <param name="vatAmount">The amount of funds to capture as value added taxes.</param>
+        /// <param name="orderItems"><seealso cref="OrderItem"/>s involved in the capture.</param>
+        /// <param name="description">A textual description of the capture.</param>
+        /// <param name="payeeReference">Transactionally unique reference from the merchant system.</param>
+        public PaymentOrderCaptureRequest(Amount amount, Amount vatAmount, List<OrderItem> orderItems, string description, string payeeReference)

```

Comment:
Why? Then the consumer must know that they have to add things them selves instead of just passing it to the constructor. I do not agree with this pattern at all.

Reply:
Because they read documentation letting them know that these properties exists and should be used if they want access to their provided functionality.